### DISCUSSION TITLE
How to deploy to a docker swarm cluster?

### DISCUSSION THREAD

**Discussion by ptitboogy** [jonaswinkler/paperless-ng#467] [Support]
Hi,

I tried to deploy paperless-ng on a swarm cluster using docker stack deploy which is running well but how can I create the superuser ?
Of course the command "docker-compose run --rm webserver createsuperuser" is not working on a swarm.
Thanks for your help.

**Comment by jonaswinkler** [comment]
Why exactly is that command not working?

**Reply by ptitboogy** [reply]
Because I deploy paperless-ng as a stack and not using the docker-compose I guess

**Reply by ptitboogy** [reply]
As a workaround, I deployed paperless-ng using docker compose, creating the superuser, removing all the services and recreate the stack using the same folders

**Reply by jonaswinkler** [reply]
I see. Are you able to run commands with docker exec on running containers?

**Reply by ptitboogy** [reply]
Didn't tried but can you explain me how can I do please ?
